### PR TITLE
Buffs the Silverscale pirates (and their species slightly), and tweaks their shuttle

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

This PR gives some buffs to the Silverscale pirates.
- Replaces their mosins with "Prime Nagants", mosins that don't jam and have a unique sprite (Thanks to NecromancerAnne)!
- Adds 4 spare Mosin clips to the ship
- Adds 1 extra box of buckshot ammo.
- Replaces the double-barreled shotgun with a hunting shotgun, which starts with slugs. The captain gets 4 extra slugs in their closet, if they feel inclined to use it. 
- Ensures all of their doors have the silicon wire cut, as intended.
- Replaces their normal suit that they all spawn with with a syndicate tactical suit (no suit sensors, 10 melee armor). (Did you know Silverscales can start with their suit sensors on? haha)
- Makes the species itself pressure immune and pierce immune.
- Gives them 1 brick of C4 and 1 med-kit of ancient supplies. 

Also goes through and touches up their ship a bit. Adds air-alarms, newscasters (because I think it would be funny), minor decal work.

## Why It's Good For The Game

The Silverscale pirates are kinda the weakest of the bunch.
What they have in flavor they lack in supplies and staying power.

- They had no extra ammunition for their mosins (WHICH COULD JAM, BIG BUMMER).
- They had no medical supplies and no easy way to treat themselves.
- They had no way to breach a high-risk area (where booty is stored)

This results in (more often than not): Silverscales run in, die. Their outline paints a big target on their forehead and their tongue gimmick, while cool, is very unreliable and ends up hurting often due to its cooldown. 

These changes seek to give the Silverscales a bit more staying power, compared to the other pirate gangs.
Additionally, it feels very thematically off that the posh lizards get cheap mosins and few supplies. They should have riches, not rags, surely. 

## Changelog

:cl: Melbert, sprites from NecromancerAnne
balance: The Silverscale species are now pierce-immune and pressure-immune (NOT temperature immune though).
balance: Beefed up the Silverscale pirate gang. Their shuttle now has extra ammo and additional general supplies. They're also now equipped with "Prime Nagant"s, which don't jam and have a unique sprite. Regal, fancy!
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
atleast they'll be a threat now, before anytime pirates get announced and someone sees that its silver-scales literally everyone stops caring

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -127,6 +128,22 @@
 	can_be_sawn_off = FALSE
 	can_jam = FALSE
 
+/obj/item/gun/ballistic/rifle/boltaction/brand_new/prime
+	name = "Regal Nagant"
```
Comment (necromanceranne) [inline code/modules/projectiles/guns/ballistic/rifle.dm:132]:
```suggestion
	name = "\improper Regal Nagant"
```

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 2]
```diff
@@ -127,6 +128,22 @@
...
+	worn_icon_state = "moistprime"
+	can_be_sawn_off = TRUE
+	sawn_desc = "An sawn-off Regal Nagant... Doing this was a sin, I hope you're happy. \
+		You are now probably one of the few people in the universe to ever hold a \"Regal Obrez\". \
+		Even thinking about that name combination makes you ill."
+
+/obj/item/gun/ballistic/rifle/boltaction/brand_new/prime/sawoff(mob/user)
+	. = ..()
+	if(.)
+		name = "Regal Obrez" // wear it loud and proud
```
Comment (necromanceranne) [inline code/modules/projectiles/guns/ballistic/rifle.dm:145]:
```suggestion
		name = "\improper Regal Obrez" // wear it loud and proud
```
Also, does the parent not call it an obrez after being sawn off? That's surprising.

Comment (MrMelbert) [reply code/modules/projectiles/guns/ballistic/rifle.dm:145]: yeah a mosin's name becomes "sawn-off Mosin Nagant"

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 3]
```diff
@@ -127,6 +128,22 @@
...
 	can_jam = FALSE
 
+/obj/item/gun/ballistic/rifle/boltaction/brand_new/prime
+	name = "Regal Nagant"
+	desc = "A prized hunting Mosin Nagant. Used for the most dangerous game."
+	icon_state = "moistprime"
+	inhand_icon_state = "moistprime"
+	worn_icon_state = "moistprime"
+	can_be_sawn_off = TRUE
+	sawn_desc = "An sawn-off Regal Nagant... Doing this was a sin, I hope you're happy. \
```
Comment (necromanceranne) [inline code/modules/projectiles/guns/ballistic/rifle.dm:138]:
```suggestion
	sawn_desc = "A sawn-off Regal Nagant... Doing this was a sin, I hope you're happy. \
```

**Comment by ATH1909** [comment]
> Makes the species itself pressure immune and pierce immune.

won't this stop them from being able to apply those meds you gave them? also, I don't see this change in the code you modified.

**Review by Imaginos16 — APPROVED** [review]
The sprites are good, nice work!

**Comment by MrMelbert** [comment]
> > Makes the species itself pressure immune and pierce immune.
> 
> won't this stop them from being able to apply those meds you gave them? also, I don't see this change in the code you modified.

1. 
image

2. Ok the fact that pierce immune species can't use bruise packs or ointment is absurd

**Review by Maurukas — APPROVED** [review]
Maps look good. Silverscales also desperately needed a boost. Or players need to stop ranking pirates by their murdering potential.

For some reason Mapdiffbot is showing all the decals being removed, but this looks fine in sDMM.

**Comment by wraith-54321** [comment]
This is very good

**Comment by Maurukas** [comment]
I'm going to make the assumption that anyone capable enough to complain about the code has had ample opportunity to do so. Mostly because I want this out of my list.

**Pull request merged**